src/packages/client/src/__tests__/integration/happy/mixed-transaction/test.ts

```diff
@@ -0,0 +1,39 @@
+import { getTestClient } from '../../../../utils/getTestClient'
+
+test('mixed transaction', async () => {
+  const PrismaClient = await getTestClient()
+  const prisma = new PrismaClient()
+
+  await prisma.user.deleteMany()
+
+  try {
+    // Test connecting and disconnecting all the time
+    const result = await prisma.$transaction([
+      prisma.user.create({
+        data: {
+          email: 'test@hey.com',
+        },
+      }),
+      prisma.user.create({
+        data: {
+          email: undefined,
+        },
+      }),
+    ])
+  } catch (e) {
+    //
+    // console.error(e)
+  }
+
+  await new Promise<void>((r) => r())
```

Comment:
What in the world is this for?

Reply:
The problem is, that without this, the current "batch" is not dispatched in the error case.